chichiwang/chichiwang.dev
# Bugfix: Allow logo to be resized

## Description
Linked to Issue: #22
The `Logo` had a fixed width/height baked into the SVG element. This PR removes it so that the consumer of the component can set the dimensions.

## Changes
* Remove the `width` and `height` attributes from the SVG element in `app/components/Logo/index.tsx`

## Steps to QA
* Pull down and switch to this branch
* Verify in browser the logo on every page (`/`, `/error`, `/notfound`) are rendered in the correct size at 250px height
* Render another `Logo` on any page at any width or height and ensure it renders as expected
